scripts/character_test.js

```diff
@@ -1,37 +1,22 @@
-/*global $ validCharacterPattern XRegExp */
+/*global validCharacterPattern XRegExp */
 /* exported testText */
 
-// regex unicode property escape is supported in Chrome and Safari (Feb 2020)
-// but not in Firefox or Edge. Use 3rd party http://xregexp.com/ instead
+// regex unicode property escape is supported in Chrome 64, Safari 11.1
+// Firefox 78, Edge 79, Opera 51. Use 3rd party http://xregexp.com/ instead
 
 // this matches any character: non-mark codepoint followed by 0 or more marks
 const charMatch = XRegExp("\\PM\\pM*", "Ag");
```

Comment:
are we aiming to get rid of all instances of XRegExp eventually, right?

Reply:
Yes.